src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -817,18 +819,14 @@ class LicenceService(
     return inFlight.isNotEmpty()
   }
 
-  private fun copyLicenceAndConditions(kind: LicenceKind, licence: EntityLicence): EntityLicence {
+  private fun copyConditions(kind: LicenceKind, licence: EntityLicence, licenceCopy: EntityLicence): EntityLicence {
```

Comment:
Non-blocking, perhaps a slight misnomer as it's also setting the status and policy version?

Reply:
yes, it does a load of stuff really. Will update in a future PR